### ISSUE TITLE
Error on building Unity project. Need to set one target to automatic

### ISSUE THREAD

**Issue by gpiffaretti** [yukiarrr/ios-build-action#18]
I'm getting an error when using this action with a Unity build. The XCode generated project has three targets, Unity-iPhone, Unity-iPhone Tests, and UnityFramework. When trying to generate a signed IPA, I'm getting the following error.
```
[00:30:01]: ▸ note: Using legacy build system
[00:30:01]: ▸ === BUILD TARGET UnityFramework OF PROJECT Unity-iPhone WITH CONFIGURATION Release ===
[00:30:01]: ▸ Check dependencies
[00:30:01]: ▸ Code Signing Error: UnityFramework does not support provisioning profiles. UnityFramework does not support provisioning profiles, but provisioning profile Apple_Dist_6-22-exp has been manually specified. Set the provisioning profile value to "Automatic" in the build settings editor.
```

Here's the configuration I'm using for the action
```
# Xcode build and generate IPA
       - uses: yukiarrr/ios-build-action@v1.1.1
         with:
           project-path: iOS/iOS/Unity-iPhone.xcodeproj
           p12-base64: ${{ secrets.BASE64_P12 }}
           mobileprovision-base64: ${{ secrets.BASE64_IOS_ADHOC_PROFILE }}
           code-signing-identity: "Apple Distribution"
           team-id: ${{ secrets.TEAM_ID }}
           scheme: "Unity-iPhone"
           output-path: ./build/iosbuild.ipa
           disable-targets: 'Unity-iPhone'
```

Here's the full log just in case. Any ideas how can I solve this to make the build succeed? Looks like it's some small tweak in configuration but I'm no expert in iOS builds and don't know exactly how to proceed.
Is there a way to set one of the targets to automatic and use the provided key and provisioning for the other targets?

```
2020-09-11T00:28:21.9362210Z [00:28:21]: [33mSeems like launching fastlane takes a while - please run[0m
2020-09-11T00:28:21.9364570Z [00:28:21]: 
2020-09-11T00:28:21.9366700Z [00:28:21]: [36m$ [sudo] gem cleanup[0m
2020-09-11T00:28:21.9369690Z [00:28:21]: 
2020-09-11T00:28:21.9377040Z [00:28:21]: [33mto uninstall outdated gems and make fastlane launch faster[0m
2020-09-11T00:28:21.9379380Z [00:28:21]: [33mAlternatively it's recommended to start using a Gemfile to lock your dependencies[0m
2020-09-11T00:28:21.9380900Z [00:28:21]: [33mTo get started with a Gemfile, run[0m
2020-09-11T00:28:21.9382030Z [00:28:21]: 
2020-09-11T00:28:21.9383480Z [00:28:21]: [36m$ bundle init[0m
2020-09-11T00:28:21.9385000Z [00:28:21]: [36m$ echo 'gem "fastlane"' >> Gemfile[0m
2020-09-11T00:28:21.9388050Z [00:28:21]: [36m$ bundle install[0m
2020-09-11T00:28:21.9389140Z [00:28:21]: 
2020-09-11T00:28:21.9390580Z [00:28:21]: [33mAfter creating the Gemfile and Gemfile.lock, commit those files into version control[0m
2020-09-11T00:28:21.9394040Z [00:28:21]: [33mGet started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile[0m
2020-09-11T00:28:33.9529930Z [00:28:33]: Sending anonymous analytics information
2020-09-11T00:28:33.9531230Z [00:28:33]: Learn more at https://docs.fastlane.tools/#metrics
2020-09-11T00:28:33.9532190Z [00:28:33]: No personal or sensitive data is sent.
2020-09-11T00:28:33.9532940Z [00:28:33]: You can disable this by adding `opt_out_usage` at the top of your Fastfile
2020-09-11T00:28:34.0963500Z [00:28:34]: [32m------------------------------[0m
2020-09-11T00:28:34.0964670Z [00:28:34]: [32m--- Step: default_platform ---[0m
2020-09-11T00:28:34.0966360Z [00:28:34]: [32m------------------------------[0m
2020-09-11T00:28:34.0971120Z [00:28:34]: [32mDriving the lane 'ios export_ipa' 🚀[0m
2020-09-11T00:28:34.0981700Z [00:28:34]: [32m-----------------------------[0m
2020-09-11T00:28:34.0982850Z [00:28:34]: [32m--- Step: create_keychain ---[0m
2020-09-11T00:28:34.0984410Z [00:28:34]: [32m-----------------------------[0m
2020-09-11T00:28:34.2916310Z [00:28:34]: [36m$ security list-keychains -d user[0m
2020-09-11T00:28:34.3022790Z [00:28:34]: ▸ [35m"/Users/runner/Library/Keychains/ios-build.keychain-db"[0m
2020-09-11T00:28:34.3024680Z [00:28:34]: [33mFound keychain '/Users/runner/Library/Keychains/ios-build.keychain-db' in list-keychains, adding to search list skipped[0m
2020-09-11T00:28:34.3030480Z [00:28:34]: [32m--------------------------------[0m
2020-09-11T00:28:34.3031630Z [00:28:34]: [32m--- Step: import_certificate ---[0m
2020-09-11T00:28:34.3032700Z [00:28:34]: [32m--------------------------------[0m
2020-09-11T00:28:34.3034760Z [00:28:34]: [36m$ security import ios-build.p12 -k '/Users/runner/Library/Keychains/ios-build.keychain-db' -P '' -T /usr/bin/codesign -T /usr/bin/security -T /usr/bin/productbuild[0m
2020-09-11T00:28:34.4068310Z [00:28:34]: ▸ [35m1 identity imported.[0m
2020-09-11T00:28:34.4221610Z [00:28:34]: Setting key partition list... (this can take a minute if there are a lot of keys installed)
2020-09-11T00:28:34.4223880Z [00:28:34]: [36m$ security set-key-partition-list -S apple-tool:,apple: -s -k 5567daac-90cc-44ff-ae80-6e1927abfcbb /Users/runner/Library/Keychains/ios-build.keychain-db 1> /dev/null[0m
2020-09-11T00:28:34.5870200Z [00:28:34]: [32m------------------------------------------[0m
2020-09-11T00:28:34.5871450Z [00:28:34]: [32m--- Step: install_provisioning_profile ---[0m
2020-09-11T00:28:34.5872580Z [00:28:34]: [32m------------------------------------------[0m
2020-09-11T00:28:34.5873300Z [00:28:34]: Installing provisioning profile...
2020-09-11T00:28:34.7880480Z [00:28:34]: [32m------------------------------------------[0m
2020-09-11T00:28:34.7882300Z [00:28:34]: [32m--- Step: update_code_signing_settings ---[0m
2020-09-11T00:28:34.7883470Z [00:28:34]: [32m------------------------------------------[0m
2020-09-11T00:28:34.7962600Z 
2020-09-11T00:28:34.7973760Z +-----------------------+------------------------------------+
2020-09-11T00:28:34.7975300Z |             [32mSummary for code signing settings[0m              |
2020-09-11T00:28:34.7976870Z +-----------------------+------------------------------------+
2020-09-11T00:28:34.7977920Z | use_automatic_signing | false                              |
2020-09-11T00:28:34.7979410Z | path                  | iOS/iOS/Unity-iPhone.xcodeproj     |
2020-09-11T00:28:34.7982430Z | targets               | ["Unity-iPhone", "UnityFramework"] |
2020-09-11T00:28:34.7983760Z +-----------------------+------------------------------------+
2020-09-11T00:28:34.7984500Z 
2020-09-11T00:28:35.3946680Z [00:28:35]: Updating the Automatic Codesigning flag to disabled for the given project '/Users/runner/work/GB_test/GB_test/iOS/iOS/Unity-iPhone.xcodeproj/project.pbxproj'
2020-09-11T00:28:35.3949820Z [00:28:35]: [33mSkipping Unity-iPhone Tests not selected (Unity-iPhone,UnityFramework)[0m
2020-09-11T00:28:35.6365770Z [00:28:35]: [32mSuccessfully updated project settings to use Code Sign Style = 'Manual'[0m
2020-09-11T00:28:35.6384990Z [00:28:35]: [32mModified Targets:[0m
2020-09-11T00:28:35.6388650Z [00:28:35]: [32m	 * Unity-iPhone[0m
2020-09-11T00:28:35.6390100Z [00:28:35]: [32m	 * UnityFramework[0m
2020-09-11T00:28:35.6396190Z [00:28:35]: [32mModified Build Configurations:[0m
2020-09-11T00:28:35.6410160Z [00:28:35]: [32m	 * Release[0m
2020-09-11T00:28:35.6447240Z [00:28:35]: [32m	 * ReleaseForProfiling[0m
2020-09-11T00:28:35.6448470Z [00:28:35]: [32m	 * ReleaseForRunning[0m
2020-09-11T00:28:35.6501690Z [00:28:35]: [32m	 * Debug[0m
2020-09-11T00:28:35.6503030Z [00:28:35]: [32m	 * Release[0m
2020-09-11T00:28:35.6506110Z [00:28:35]: [32m	 * ReleaseForProfiling[0m
2020-09-11T00:28:35.6507750Z [00:28:35]: [32m	 * ReleaseForRunning[0m
2020-09-11T00:28:35.6509120Z [00:28:35]: [32m	 * Debug[0m
2020-09-11T00:28:35.6510520Z [00:28:35]: [32m-----------------------------------------[0m
2020-09-11T00:28:35.6512000Z [00:28:35]: [32m--- Step: update_project_provisioning ---[0m
2020-09-11T00:28:35.6513440Z [00:28:35]: [32m-----------------------------------------[0m
2020-09-11T00:28:35.6515160Z [00:28:35]: You’re updating provisioning profiles directly in your project, but have you considered easier ways to do code signing?
2020-09-11T00:28:35.6516450Z [00:28:35]: https://docs.fastlane.tools/codesigning/GettingStarted/
2020-09-11T00:28:35.6518060Z [00:28:35]: Downloading root certificate from (https://www.apple.com/appleca/AppleIncRootCertificate.cer) to path '/tmp/AppleIncRootCertificate.cer'
2020-09-11T00:28:35.8339570Z [00:28:35]: Parsing mobile provisioning profile from 'ios-build.mobileprovision'
2020-09-11T00:28:35.8442010Z [00:28:35]: [32mGoing to update project 'iOS/iOS/Unity-iPhone.xcodeproj' with UUID[0m
2020-09-11T00:28:35.9119500Z [00:28:35]: [32mUpdating target Unity-iPhone...[0m
2020-09-11T00:28:35.9121160Z [00:28:35]: [32mUpdating configuration Release...[0m
2020-09-11T00:28:35.9133420Z [00:28:35]: [32mUpdating configuration ReleaseForProfiling...[0m
2020-09-11T00:28:35.9135700Z [00:28:35]: [32mUpdating configuration ReleaseForRunning...[0m
2020-09-11T00:28:35.9137490Z [00:28:35]: [32mUpdating configuration Debug...[0m
2020-09-11T00:28:35.9138820Z [00:28:35]: [32mUpdating target Unity-iPhone Tests...[0m
2020-09-11T00:28:35.9139970Z [00:28:35]: [32mUpdating configuration Release...[0m
2020-09-11T00:28:35.9141150Z [00:28:35]: [32mUpdating configuration ReleaseForProfiling...[0m
2020-09-11T00:28:35.9142380Z [00:28:35]: [32mUpdating configuration ReleaseForRunning...[0m
2020-09-11T00:28:35.9143550Z [00:28:35]: [32mUpdating configuration Debug...[0m
2020-09-11T00:28:35.9144710Z [00:28:35]: [32mUpdating target UnityFramework...[0m
2020-09-11T00:28:35.9145940Z [00:28:35]: [32mUpdating configuration Release...[0m
2020-09-11T00:28:35.9147210Z [00:28:35]: [32mUpdating configuration ReleaseForProfiling...[0m
2020-09-11T00:28:35.9148400Z [00:28:35]: [32mUpdating configuration ReleaseForRunning...[0m
2020-09-11T00:28:35.9149560Z [00:28:35]: [32mUpdating configuration Debug...[0m
2020-09-11T00:28:36.0476520Z [00:28:36]: [32mSuccessfully updated project settings in 'iOS/iOS/Unity-iPhone.xcodeproj'[0m
2020-09-11T00:28:36.0482270Z [00:28:36]: [32m---------------------------------[0m
2020-09-11T00:28:36.0483530Z [00:28:36]: [32m--- Step: update_project_team ---[0m
2020-09-11T00:28:36.0484440Z [00:28:36]: [32m---------------------------------[0m
2020-09-11T00:28:36.1373550Z [00:28:36]: Updating development team (***) for target `Unity-iPhone` in the project 'iOS/iOS/Unity-iPhone.xcodeproj'
2020-09-11T00:28:36.3158200Z [00:28:36]: [32mSuccessfully updated project settings to use Developer Team ID '***' for target `Unity-iPhone`[0m
2020-09-11T00:28:36.3160430Z [00:28:36]: Updating development team (***) for target `Unity-iPhone Tests` in the project 'iOS/iOS/Unity-iPhone.xcodeproj'
2020-09-11T00:28:36.4330280Z [00:28:36]: [32mSuccessfully updated project settings to use Developer Team ID '***' for target `Unity-iPhone Tests`[0m
2020-09-11T00:28:36.4331800Z [00:28:36]: Updating development team (***) for target `UnityFramework` in the project 'iOS/iOS/Unity-iPhone.xcodeproj'
2020-09-11T00:28:36.5536300Z [00:28:36]: [32mSuccessfully updated project settings to use Developer Team ID '***' for target `UnityFramework`[0m
2020-09-11T00:28:38.1030140Z [00:28:38]: [32m-----------------------[0m
2020-09-11T00:28:38.1031180Z [00:28:38]: [32m--- Step: build_app ---[0m
2020-09-11T00:28:38.1032090Z [00:28:38]: [32m-----------------------[0m
2020-09-11T00:28:38.1090700Z [00:28:38]: [33mResolving Swift Package Manager dependencies...[0m
2020-09-11T00:28:38.1092170Z [00:28:38]: [36m$ xcodebuild -resolvePackageDependencies -scheme Unity-iPhone -project iOS/iOS/Unity-iPhone.xcodeproj -configuration Release[0m
2020-09-11T00:29:18.2319290Z [00:29:18]: ▸ [35mCommand line invocation:[0m
2020-09-11T00:29:18.2321030Z [00:29:18]: ▸ [35m    /Applications/Xcode_11.7.app/Contents/Developer/usr/bin/xcodebuild -resolvePackageDependencies -scheme Unity-iPhone -project iOS/iOS/Unity-iPhone.xcodeproj -configuration Release[0m
2020-09-11T00:29:23.8790020Z [00:29:23]: ▸ [35mresolved source packages: [0m
2020-09-11T00:29:23.8950080Z [00:29:23]: [36m$ xcodebuild -showBuildSettings -scheme Unity-iPhone -project iOS/iOS/Unity-iPhone.xcodeproj -configuration Release[0m
2020-09-11T00:29:26.9106120Z [00:29:26]: [33mCommand timed out after 3 seconds on try 1 of 4, trying again with a 6 second timeout...[0m
2020-09-11T00:29:32.9252790Z [00:29:32]: [33mCommand timed out after 6 seconds on try 2 of 4, trying again with a 12 second timeout...[0m
2020-09-11T00:29:43.3649310Z [00:29:43]: Detected provisioning profile mapping: {:"com.FSStudio.${PRODUCT_NAME:rfc1123identifier}"=>"Apple_Dist_6-22-exp", :"com.unity3d.framework"=>"Apple_Dist_6-22-exp"}
2020-09-11T00:29:43.7671110Z 
2020-09-11T00:29:43.7712000Z +------------------------------------------------------------------------------------+-----------------------------------------------------------+
2020-09-11T00:29:43.7713840Z |                                                            [32mSummary for gym 2.157.2[0m                                                             |
2020-09-11T00:29:43.7715560Z +------------------------------------------------------------------------------------+-----------------------------------------------------------+
2020-09-11T00:29:43.7716720Z | project                                                                            | iOS/iOS/Unity-iPhone.xcodeproj                            |
2020-09-11T00:29:43.7717470Z | configuration                                                                      | Release                                                   |
2020-09-11T00:29:43.7718620Z | scheme                                                                             | Unity-iPhone                                              |
2020-09-11T00:29:43.7719510Z | output_name                                                                        | ._build_iosbuild                                          |
2020-09-11T00:29:43.7720280Z | clean                                                                              | true                                                      |
2020-09-11T00:29:43.7721870Z | export_method                                                                      | app-store                                                 |
2020-09-11T00:29:43.7722650Z | destination                                                                        | generic/platform=iOS                                      |
2020-09-11T00:29:43.7723940Z | export_options.provisioningProfiles.com.FSStudio.${PRODUCT_NAME:rfc1123identifier} | Apple_Dist_6-22-exp                                       |
2020-09-11T00:29:43.7725100Z | export_options.provisioningProfiles.com.unity3d.framework                          | Apple_Dist_6-22-exp                                       |
2020-09-11T00:29:43.7726240Z | build_path                                                                         | /Users/runner/Library/Developer/Xcode/Archives/2020-09-11 |
2020-09-11T00:29:43.7726950Z | output_directory                                                                   | .                                                         |
2020-09-11T00:29:43.7727620Z | silent                                                                             | false                                                     |
2020-09-11T00:29:43.7728290Z | skip_package_ipa                                                                   | false                                                     |
2020-09-11T00:29:43.7728970Z | skip_package_pkg                                                                   | false                                                     |
2020-09-11T00:29:43.7729910Z | result_bundle                                                                      | false                                                     |
2020-09-11T00:29:43.7730680Z | buildlog_path                                                                      | ~/Library/Logs/gym                                        |
2020-09-11T00:29:43.7731370Z | skip_profile_detection                                                             | false                                                     |
2020-09-11T00:29:43.7732050Z | xcode_path                                                                         | /Applications/Xcode_11.7.app                              |
2020-09-11T00:29:43.7733270Z +------------------------------------------------------------------------------------+-----------------------------------------------------------+
2020-09-11T00:29:43.7733640Z 
2020-09-11T00:29:43.7738830Z [00:29:43]: [36m$ set -o pipefail && xcodebuild -scheme Unity-iPhone -project iOS/iOS/Unity-iPhone.xcodeproj -configuration Release -destination 'generic/platform=iOS' -archivePath /Users/runner/Library/Developer/Xcode/Archives/2020-09-11/._build_iosbuild\ 2020-09-11\ 00.29.43.xcarchive clean archive | tee /Users/runner/Library/Logs/gym/GBAR-Unity-iPhone.log | xcpretty[0m
2020-09-11T00:30:00.0972510Z [00:30:00]: ▸ [35m** ARCHIVE FAILED **[0m
2020-09-11T00:30:00.0976180Z [00:30:00]: ▸ [35mThe following build commands failed:[0m
2020-09-11T00:30:00.0977230Z [00:30:00]: ▸ [35m	Check dependencies[0m
2020-09-11T00:30:00.0978150Z [00:30:00]: ▸ [35m(1 failure)[0m
2020-09-11T00:30:00.9665030Z [00:30:00]: [35m[33m▸[0m [39;1mCleaning[0m Unity-iPhone/UnityFramework [Release][0m
2020-09-11T00:30:00.9666220Z [00:30:00]: [35m[33m▸[0m [39;1mCheck Dependencies[0m
2020-09-11T00:30:00.9679620Z [00:30:00]: [35m[33m▸[0m [39;1mCleaning[0m Unity-iPhone/Unity-iPhone [Release][0m
2020-09-11T00:30:00.9680680Z [00:30:00]: [35m[33m▸[0m [39;1mCheck Dependencies[0m
2020-09-11T00:30:00.9686000Z [00:30:00]: [35m[33m▸[0m [39;1mClean[0m Succeeded[0m
2020-09-11T00:30:00.9687570Z [00:30:00]: [35m[33m▸[0m [39;1mBuilding[0m Unity-iPhone/UnityFramework [Release][0m
2020-09-11T00:30:00.9689240Z [00:30:00]: [35m[33m▸[0m [39;1mCheck Dependencies[0m
2020-09-11T00:30:00.9749280Z ** ARCHIVE FAILED **
2020-09-11T00:30:00.9749730Z 
2020-09-11T00:30:00.9750030Z 
2020-09-11T00:30:00.9750530Z The following build commands failed:
2020-09-11T00:30:00.9751040Z 	Check dependencies
2020-09-11T00:30:00.9751510Z (1 failure)
2020-09-11T00:30:00.9752750Z [33m▸[0m [39;1mCleaning[0m Unity-iPhone/UnityFramework [Release]
2020-09-11T00:30:00.9754040Z [33m▸[0m [39;1mCheck Dependencies[0m
2020-09-11T00:30:00.9755130Z [33m▸[0m [39;1mCleaning[0m Unity-iPhone/Unity-iPhone [Release]
2020-09-11T00:30:00.9756060Z [33m▸[0m [39;1mCheck Dependencies[0m
2020-09-11T00:30:00.9757040Z [33m▸[0m [39;1mClean[0m Succeeded
2020-09-11T00:30:00.9758000Z [33m▸[0m [39;1mBuilding[0m Unity-iPhone/UnityFramework [Release]
2020-09-11T00:30:00.9758940Z [33m▸[0m [39;1mCheck Dependencies[0m
2020-09-11T00:30:00.9759860Z [00:30:00]: [31mExit status: 65[0m
2020-09-11T00:30:00.9822230Z [00:30:00]: 
2020-09-11T00:30:00.9823840Z [00:30:00]: [33mMaybe the error shown is caused by using the wrong version of Xcode[0m
2020-09-11T00:30:00.9825160Z [00:30:00]: [33mFound multiple versions of Xcode in '/Applications/'[0m
2020-09-11T00:30:00.9826780Z [00:30:00]: [33mMake sure you selected the right version for your project[0m
2020-09-11T00:30:00.9828470Z [00:30:00]: [33mThis build process was executed using '/Applications/Xcode_11.7.app'[0m
2020-09-11T00:30:00.9829820Z [00:30:00]: [33mIf you want to update your Xcode path, either[0m
2020-09-11T00:30:00.9830420Z [00:30:00]: 
2020-09-11T00:30:00.9832280Z [00:30:00]: - Specify the Xcode version in your Fastfile
2020-09-11T00:30:00.9833840Z [00:30:00]: ▸ [35mxcversion(version: "8.1") # Selects Xcode 8.1.0[0m
2020-09-11T00:30:00.9834450Z [00:30:00]: 
2020-09-11T00:30:00.9835790Z [00:30:00]: - Specify an absolute path to your Xcode installation in your Fastfile
2020-09-11T00:30:00.9837170Z [00:30:00]: ▸ [35mxcode_select "/Applications/Xcode8.app"[0m
2020-09-11T00:30:00.9837800Z [00:30:00]: 
2020-09-11T00:30:00.9838980Z [00:30:00]: - Manually update the path using
2020-09-11T00:30:00.9841200Z [00:30:00]: ▸ [35msudo xcode-select -s /Applications/Xcode.app[0m
2020-09-11T00:30:00.9841810Z [00:30:00]: 
2020-09-11T00:30:00.9980180Z 
2020-09-11T00:30:00.9990000Z +---------------+------------------------------+
2020-09-11T00:30:00.9991060Z |              [32m[33mBuild environment[0m               |
2020-09-11T00:30:00.9993220Z +---------------+------------------------------+
2020-09-11T00:30:00.9993810Z | xcode_path    | /Applications/Xcode_11.7.app |
2020-09-11T00:30:00.9994360Z | gym_version   | 2.157.2                      |
2020-09-11T00:30:00.9995290Z | export_method | app-store                    |
2020-09-11T00:30:00.9995850Z | sdk           | iPhoneOS13.7.sdk             |
2020-09-11T00:30:00.9996760Z +---------------+------------------------------+
2020-09-11T00:30:00.9997080Z 
2020-09-11T00:30:01.0006870Z [00:30:01]: ▸ [35mnote: Using legacy build system[0m
2020-09-11T00:30:01.0008530Z [00:30:01]: ▸ [35m=== BUILD TARGET UnityFramework OF PROJECT Unity-iPhone WITH CONFIGURATION Release ===[0m
2020-09-11T00:30:01.0010050Z [00:30:01]: ▸ [35mCheck dependencies[0m
2020-09-11T00:30:01.0012070Z [00:30:01]: ▸ [35mCode Signing Error: UnityFramework does not support provisioning profiles. UnityFramework does not support provisioning profiles, but provisioning profile Apple_Dist_6-22-exp has been manually specified. Set the provisioning profile value to "Automatic" in the build settings editor.[0m
2020-09-11T00:30:01.0013130Z [00:30:01]: 
2020-09-11T00:30:01.0014840Z [00:30:01]: [31m⬆️  Check out the few lines of raw `xcodebuild` output above for potential hints on how to solve this error[0m
2020-09-11T00:30:01.0016710Z [00:30:01]: [33m📋  For the complete and more detailed error log, check the full log at:[0m
2020-09-11T00:30:01.0018090Z [00:30:01]: [33m📋  /Users/runner/Library/Logs/gym/GBAR-Unity-iPhone.log[0m
2020-09-11T00:30:01.0018720Z [00:30:01]: 
2020-09-11T00:30:01.0020310Z [00:30:01]: [31mLooks like fastlane ran into a build/archive error with your project[0m
2020-09-11T00:30:01.0021650Z [00:30:01]: [31mIt's hard to tell what's causing the error, so we wrote some guides on how[0m
2020-09-11T00:30:01.0022940Z [00:30:01]: [31mto troubleshoot build and signing issues: https://docs.fastlane.tools/codesigning/getting-started/[0m
2020-09-11T00:30:01.0024070Z [00:30:01]: [31mBefore submitting an issue on GitHub, please follow the guide above and make[0m
2020-09-11T00:30:01.0025130Z [00:30:01]: [31msure your project is set up correctly.[0m
2020-09-11T00:30:01.0026200Z [00:30:01]: [31mfastlane uses `xcodebuild` commands to generate your binary, you can see the[0m
2020-09-11T00:30:01.0027280Z [00:30:01]: [31mthe full commands printed out in yellow in the above log.[0m
2020-09-11T00:30:01.0028380Z [00:30:01]: [31mMake sure to inspect the output above, as usually you'll find more error information there[0m
2020-09-11T00:30:01.0028990Z [00:30:01]: 
2020-09-11T00:30:01.0099240Z +---------------------------+-----------------------------------------------------+
2020-09-11T00:30:01.0100610Z |                                  [33mLane Context[0m                                   |
2020-09-11T00:30:01.0101830Z +---------------------------+-----------------------------------------------------+
2020-09-11T00:30:01.0102500Z | DEFAULT_PLATFORM          | ios                                                 |
2020-09-11T00:30:01.0103170Z | PLATFORM_NAME             | ios                                                 |
2020-09-11T00:30:01.0103810Z | LANE_NAME                 | ios export_ipa                                      |
2020-09-11T00:30:01.0104880Z | KEYCHAIN_PATH             | ~/Library/Keychains/ios-build.keychain              |
2020-09-11T00:30:01.0106540Z | ORIGINAL_DEFAULT_KEYCHAIN | "/Users/runner/Library/Keychains/login.keychain-db" |
2020-09-11T00:30:01.0107700Z +---------------------------+-----------------------------------------------------+
2020-09-11T00:30:01.0108740Z [00:30:01]: [31mError building the application - see the log above[0m
2020-09-11T00:30:01.3312550Z 
2020-09-11T00:30:01.3331010Z +------+------------------------------+-------------+
2020-09-11T00:30:01.3334000Z |                 [32mfastlane summary[0m                  |
2020-09-11T00:30:01.3335100Z +------+------------------------------+-------------+
2020-09-11T00:30:01.3335850Z | Step | Action                       | Time (in s) |
2020-09-11T00:30:01.3336860Z +------+------------------------------+-------------+
2020-09-11T00:30:01.3337440Z | 1    | default_platform             | 0           |
2020-09-11T00:30:01.3338010Z | 2    | create_keychain              | 0           |
2020-09-11T00:30:01.3338580Z | 3    | import_certificate           | 0           |
2020-09-11T00:30:01.3339140Z | 4    | install_provisioning_profile | 0           |
2020-09-11T00:30:01.3339700Z | 5    | update_code_signing_settings | 0           |
2020-09-11T00:30:01.3340260Z | 6    | update_project_provisioning  | 0           |
2020-09-11T00:30:01.3340970Z | 7    | update_project_team          | 0           |
2020-09-11T00:30:01.3342480Z | 💥   | [31mbuild_app[0m                    | 82          |
2020-09-11T00:30:01.3343880Z +------+------------------------------+-------------+
2020-09-11T00:30:01.3344200Z 
2020-09-11T00:30:01.3345120Z [00:30:01]: [31mfastlane finished with errors[0m
2020-09-11T00:30:01.3346080Z [31m
2020-09-11T00:30:01.3346960Z [!] Error building the application - see the log above[0m
2020-09-11T00:30:01.3347290Z 
2020-09-11T00:30:01.3347830Z #######################################################################
2020-09-11T00:30:01.3348410Z # fastlane 2.159.0 is available. You are on 2.157.2.
2020-09-11T00:30:01.3348980Z # You should use the latest version.
2020-09-11T00:30:01.3349540Z # Please update using `sudo gem install fastlane`.
2020-09-11T00:30:01.3350140Z #######################################################################
2020-09-11T00:30:01.8014610Z ##[error]The process '/bin/bash' failed with exit code 1
```

**Comment by gpiffaretti** [comment]
I forked the repo and made a quick fix specific for Unity projects. Looks like we'd need more parameters exposed in the action in order to achieve this.
https://github.com/gpiffaretti/ios-build-action/commit/c27b0f24dd229827647729067d2595cf53153b67

**Comment by yukiarrr** [comment]
@gpiffaretti Thanks for forking and fixing repo!

Looking at your fix, I thought it would be good to use `disable_targets` as targets.

```diff
    update_project_provisioning(
      xcodeproj: ENV["PROJECT_PATH"],
      profile: "ios-build.mobileprovision",
      code_signing_identity: ENV["CODE_SIGNING_IDENTITY"],
+     target_filter: disable_targets&.join("|")
    )
    update_project_team(
      path: ENV["PROJECT_PATH"],
      teamid: ENV["TEAM_ID"],
+     targets: disable_targets
    )
```

Can you create Pull Request?

**Comment by yukiarrr** [comment]
@gpiffaretti 
> Can you create Pull Request?

Is the above possible?

**Comment by yukiarrr** [comment]
Including the PR that resolved this issue, v1.2.0 has been released.
Thanks for your report!
